Scripts/SAPSetupAssistantValidationScripts/SetupAssistantSAPValidateGateway.ps1

```diff
@@ -0,0 +1,73 @@
+<#
+.SYNOPSIS
+    Checks for On-permises data gateway installation.
+.DESCRIPTION
+    This script attempts to
+    1. Check if the On-permises data gateway is installed and validates the
```

Comment:
Should be "on-premises", not "On-permises"

Reply:
Updated

Reply:
There are still misspellings on line 3 and 6 - should be "on-premises"